server/bleep/src/webserver/answer.rs

```diff
@@ -685,6 +688,25 @@ Answer:",
     }
 }
 
+async fn github_user_id(app: &Application) -> Option<String> {
```

Comment:
Can you take a look if this exists in some form or fashion? I'd also prefer if this was closer to the other code dealing with github remotes, e.g `remotes::github::current_user(&app)` or something similar.